libraries/stdlib/jvm/src/kotlin/io/files/FileTreeWalk.kt

```diff
@@ -92,14 +113,14 @@ public class FileTreeWalk private constructor(
             }
         }
 
-        private tailrec fun gotoNext(): File? {
+        private tailrec fun gotoNextDfs(): File? {
```

Comment:
Again, I'd prefer to spell out `Dfs`, and likewise for `Bfs` below.

Reply:
Agree.

Reply:
> Again, I'd prefer to spell out `Dfs`, and likewise for `Bfs` below.

Done.